e2e/node/acp.test.ts

```diff
@@ -62,28 +70,41 @@ describe("An ACP Solid server", () => {
   let options: { fetch: typeof global.fetch };
   let session: Session;
   let sessionResource: string;
+  let sessionContainer: string;
 
   beforeEach(async () => {
     session = await getAuthenticatedSession(env);
-    sessionResource = `${env.pod}${sessionResourcePrefix}${session.info.sessionId}`;
-    options = { fetch: session.fetch };
-    await saveSolidDatasetAt(sessionResource, createSolidDataset(), options);
+    const { containerUrl, resourceUrl, fetchWithAgent } =
+      await setupTestResources(session, TEST_SLUG, env.pod);
+    sessionResource = resourceUrl;
+    sessionContainer = containerUrl;
+    options = { fetch: fetchWithAgent };
```

Comment:
might be better to call this `fetchOptions` as that's what the type is called, might be clearer than just `options`